bedrock/3_EntityWorkflow/run_worker.py

```diff
@@ -0,0 +1,33 @@
+import asyncio
+import concurrent.futures
+import logging
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+from activities import prompt_bedrock
+from workflows import EntityBedrockWorkflow
+
+
+async def main():
+    # Create client connected to server at the given address
```

Comment:
Inconsistent with whether you capitalize your comments and inconsistent with whether you add comments above client connect